tools/bridge/end2end-tests/README.md

```diff
@@ -1,24 +1,27 @@
 # Bridge end2end tests
 
-These are the end2tests of the Trustlines Blockchain Bridge (tlbc-bridge).
-
+This are the end2end tests of the Trustlines Blockchain Bridge (tlbc-bridge).
```

Comment:
I'd suggest to write 'This directory contains the end2test tests...'

Reply:
:heavy_check_mark: